tool/gravity/cli/status.go

```diff
@@ -191,17 +191,21 @@ func printStatus(operator ops.Operator, status clusterStatus, printOptions print
 
 	case printOptions.quiet:
 	default:
-		return trace.Wrap(printStatusWithOptions(status, printOptions))
+		site, err := operator.GetLocalSite()
```

Comment:
Why do you need to retrieve the cluster object again, you already added the provider field to the status, no?